core/modules/admin_bar/css/admin_bar.css

```diff
@@ -60,174 +60,28 @@
-  background-size: 16px;
-}
-#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-devel {
-  background: transparent url(../images/code--white--64.png) 5% center no-repeat;
-  background-size: 16px;
-}
-#admin-bar #admin-bar-menu > li > .dropdown > li > a.admin-reports {
-  background: transparent url(../images/info-circle--white--64.png) 5% center no-repeat;
-  background-size: 16px;
-}
-[dir="rtl"] #admin-bar #admin-bar-menu > li > .dropdown > li > a {
-  background-position: 95% center;
-}
-
-/* Admin tasks. */
-#admin-bar #admin-bar-extra.dropdown li > span.extra {
-  background: transparent url(../images/clipboard-check--white--64.png) 5% center no-repeat; /* LTR */
-  padding-left: 28px; /* LTR */
-  background-size: 16px;
-}
-[dir="rtl"] #admin-bar #admin-bar-extra.dropdown li > span.extra {
-  background-position: 95% center;
-  padding-left: 10px;
-  padding-right: 28px;
-}
-
-/* Locale link in Admin bar */
-#admin-bar .dropdown .admin-bar-locale-links > span {
-  background: transparent url(../images/language--white--64.png) 95% center no-repeat;
-  background-size: 20px;
-  padding-right: 30px; /* LTR */
-}
-
-[dir="rtl"] #admin-bar .dropdown .admin-bar-locale-links > span {
-  background-position: 5% center;
-  padding-right: 10px;
-  padding-left: 30px;
-}
-
-/* Page link. */
-#admin-bar .dropdown .admin-bar-page-links > span {
-  background: transparent url(../images/page--white--64.png) 90% center no-repeat;
-  background-size: 14px;
-  padding-right: 30px; /* LTR */
-}
-[dir="rtl"] #admin-bar .dropdown .admin-bar-page-links > span {
-  background-position: 10% center;
-  padding-right: 10px;
-  padding-left: 30px;
-}
-
-/* Logged in users. */
-#admin-bar .dropdown .admin-bar-users > a {
-  background: transparent url(../images/user-friends--white--64.png) 90% center no-repeat;
-  background-size: 16px;
-  padding-right: 28px; /* LTR */
-}
-[dir="rtl"] #admin-bar .dropdown .admin-bar-users > a {
-  background-position: 10% center;
-  padding-right: 10px;
-  padding-left: 28px;
-}
-
-/* My Account link. */
-#admin-bar .dropdown .admin-bar-account > a {
-  background: transparent url(../images/user-circle--white--64.png) 90% center no-repeat; /* LTR */
-  padding-right: 28px; /* LTR */
-  background-size: 16px;
-}
-[dir="rtl"] #admin-bar .dropdown .admin-bar-account > a {
-  background-position: 10% center;
-  padding-right: 10px;
-  padding-left: 28px;
-}
-
-/* Log Out link. */
-#admin-bar .dropdown .admin-bar-logout > a {
-  background: transparent url(../images/sign-out--white--64.png) 90% center no-repeat; /* LTR */
-  background-size: 16px;
-  padding-right: 28px; /* LTR */
-}
-[dir="rtl"] #admin-bar .dropdown .admin-bar-logout > a {
-  background-position: 10% center;
-  padding-right: 10px;
-  padding-left: 28px;
+#admin-bar svg.icon {
+  width: 2em;
+  height: 2em;
+  margin: -0.2em 0;
+  padding-right: 0.5em;
+  box-sizing: content-box;
+  vertical-align: bottom;
```

Comment:
Long shot, but if using `vertical-align: middle` instead, the negative margin above might be obsolete.